library_samples/Python3/ocs_sample_library_preview/DataView/DataView.py

```diff
@@ -114,80 +127,128 @@ def Queries(self, queries):
         self.__queries = queries
 
     @property
-    def Mappings(self):
+    def FieldSets(self):
         """
-        array of DataViewmapping   not required
+
         :return:
         """
-        return self.__mappings
+        return self.__fieldsets
 
-    @Mappings.setter
-    def Mappings(self, mappings):
+    @FieldSets.setter
+    def FieldSets(self, fieldsets):
         """
-        array of DataViewmapping   not required
-        :param mappings:
+        :param fieldsets:
         :return:
         """
-        self.__mappings = mappings
+        self.__fieldsets = fieldsets
 
     @property
-    def IndexConfig(self):
+    def Sectioners(self):
         """
-        DataViewindexConfig   not required
+           not required
         :return:
         """
-        return self.__indexConfig
+        return self.__sectioners
 
-    @IndexConfig.setter
-    def IndexConfig(self, indexConfig):
+    @Sectioners.setter
+    def Sectioners(self, sectioners):
         """
         DataViewindexConfig   not required
         :param indexConfig:
         :return:
         """
-        self.__indexConfig = indexConfig
+        self.__sectioners = sectioners
 
     @property
-    def IndexDataType(self):
+    def IndexTypeCode(self):
         """
-        Currently limited to "DateTime"   required
         :return:
         """
-        return self.__indexDataType
+        return self.__indextypecode
 
-    @IndexDataType.setter
-    def IndexDataType(self, indexDataType):
+    @IndexTypeCode.setter
+    def IndexTypeCode(self, indextypecode):
         """
         Currently limited to "DateTime"   required
         :param indexDataType:
         :return:
         """
-        self.__indexDataType = indexDataType
+        self.__indextypecode = indextypecode
 
     @property
-    def GroupRules(self):
+    def DefaultStartIndex(self):
         """
         Array of DataViewGroupRule   not required
         :return:
         """
-        return self.__groupRules
+        return self.__defaultstartindex
+
+    @DefaultStartIndex.setter
+    def DefaultStartIndex(self, defaultstartindex):
+        """DefaultStartIndex
+        Array of DataViewGroupRule   not required
```

Comment:
Copy/paste error, no param `groupRules`, and param `defaultstartindex` is not described